assets/src/edit-story/components/checklist/checks/videoOptimization.js

```diff
@@ -218,7 +232,7 @@ export const BulkVideoOptimization = () => {
         thumbnailCount={unoptimizedVideos.length}
         thumbnail={unoptimizedVideos.map((element) => (
           <Thumbnail
-            key={element.id}
+            key={element.resource.id}
```

Comment:
any reason to confirm that resource id is available? `element?.resource.id`

Reply:
since we're selecting video types the `resource` property should always be available on unoptimized videos